package.json

```diff
@@ -34,100 +34,106 @@
+    "@release-it-plugins/lerna-changelog": "^7.0.0",
+    "@tsconfig/ember": "^3.0.8",
+    "@types/ember": "^4.0.11",
+    "@types/ember__application": "^4.0.11",
+    "@types/ember__array": "^4.0.10",
+    "@types/ember__component": "^4.0.22",
+    "@types/ember__controller": "^4.0.12",
+    "@types/ember__debug": "^4.0.8",
+    "@types/ember__destroyable": "^4.0.5",
+    "@types/ember__engine": "^4.0.11",
+    "@types/ember__error": "^4.0.6",
+    "@types/ember__modifier": "^4.0.9",
+    "@types/ember__object": "^4.0.12",
+    "@types/ember__polyfills": "^4.0.6",
+    "@types/ember__routing": "^4.0.22",
+    "@types/ember__runloop": "^4.0.10",
+    "@types/ember__service": "^4.0.9",
+    "@types/ember__string": "^3.16.3",
+    "@types/ember__template": "^4.0.7",
+    "@types/ember__test": "^4.0.6",
+    "@types/ember__test-helpers": "^2.9.3",
+    "@types/ember__utils": "^4.0.7",
+    "@types/ember-data": "^4.4.16",
+    "@types/ember-data__adapter": "^4.0.6",
+    "@types/ember-data__model": "^4.0.5",
+    "@types/ember-data__serializer": "^4.0.6",
+    "@types/ember-data__store": "^4.0.7",
+    "@types/ember-qunit": "^6.1.3",
+    "@types/ember-resolver": "^9.0.2",
+    "@types/node": "^22.10.7",
+    "@types/qunit": "^2.19.12",
+    "@types/rsvp": "^4.0.9",
+    "@typescript-eslint/eslint-plugin": "^8.20.0",
+    "@typescript-eslint/parser": "^8.20.0",
     "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.0.1",
-    "date-fns": "^2.29.3",
-    "ember-auto-import": "^2.6.3",
-    "ember-cli": "~4.12.1",
-    "ember-cli-app-version": "^6.0.0",
+    "concurrently": "^8.2.2",
+    "date-fns": "^4.1.0",
+    "ember-auto-import": "^2.10.0",
+    "ember-cli": "~5.12.0",
+    "ember-cli-app-version": "^7.0.0",
     "ember-cli-autoprefixer": "^2.0.0",
-    "ember-cli-babel": "^7.26.11",
-    "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-dependency-checker": "^3.3.3",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sass": "^11.0.1",
-    "ember-cli-typescript": "^5.2.1",
-    "ember-concurrency": "^3.1.0",
-    "ember-data": "~4.12.2",
-    "ember-focus-trap": "^1.0.2",
-    "ember-load-initializers": "^2.1.2",
-    "ember-modifier": "^4.1.0",
-    "ember-page-title": "^7.0.0",
-    "ember-plausible": "^0.2.0",
-    "ember-power-select": "^7.1.0",
-    "ember-qunit": "^6.2.0",
-    "ember-resolver": "^10.0.0",
-    "ember-resources": "^6.3.1",
-    "ember-source": "~4.12.0",
-    "ember-template-lint": "^5.7.2",
-    "ember-test-selectors": "^6.0.0",
+    "ember-concurrency": "^3.1.1",
+    "ember-data": "~5.3.9",
+    "ember-focus-trap": "^1.1.1",
+    "ember-load-initializers": "^3.0.1",
+    "ember-modifier": "^4.2.0",
+    "ember-modify-based-class-resource": "^1.1.1",
+    "ember-page-title": "^8.2.3",
+    "ember-plausible": "^0.4.0",
+    "ember-power-select": "^7.2.0",
+    "ember-qunit": "^9.0.1",
```

Comment:
Hmm, I think this might be the cause of the test faillure in CI.

I'm guessing you bumped this to v9 manually? I think you either need to downgrade to v8, or do something like this to make it work in v9: https://github.com/ember-cli/ember-cli/commit/898ecfec75f804aa3dedf8bc33aa011b6087028e